docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -51,6 +58,11 @@ export JAVA_HOME=<jre_installation_folder>
 3. Use the **gene** table if you query information such as hugo symbols, types of the gene 
 4. Use **reference_genome_gene** table if you query information such as chromosome, cytoband, exonic length, or the start or end of the gene
 5. Load genes only to the **reference_genome_gene** table without updating the **gene** table, please use the following command:
```

Comment:
@yichaoS I don't really understand why someone would be doing this - but if they want to this procedure would remain the same as it was before so this is correct.

Reply:
I don't understand this mode of operation either (what it does and why)